README.md

```diff
@@ -84,40 +85,44 @@ weave launch-router \
   $KUBE_WORKER_1 $KUBE_WORKER_2
 ```
 
-If you only know some of the peer hostnames/IPs, you can launch Weave Net router like this:
+If only some of the peer hostnames/IPs are known, launch the Weave Net router as follows:
 
 ```Shell
 weave launch-router --init-peer-count 6 $KUBE_ETCD_1 $KUBE_MASTER_0 $KUBE_WORKER_1
 ```
 
-or even
+or:
 
 ```Shell
 weave launch-router --init-peer-count 6 $KUBE_MASTER_0
 ```
 
-Otherwise, you could also run `weave connect` later, just make sure to pass `--init-peer-count` with a number of servers you are expecing to have in your cluster.
+You can also run `weave connect` at a later time, and pass `--init-peer-count` with the number of servers you expect to have in your cluster.
 
-Next, you need to launch Weave proxy for the Docker API. It's crictical to pass `--rewrite-inspect` flag for Kubernetes integration to function properly.
+In order to communicate with the docker daemon through Weave, launch Weave Docker API Proxy. 
 
 ```Shell
 weave launch-proxy --rewrite-inspect
 ```
 
-And finally, you need to expose the host on Weave Net and set a DNS record for it like so:
+>>**Important** For the Kubernetes integration to function properly, it is crictical that you pass the `--rewrite-inspect` flag when launching the Weave Docker API Proxy.
+
+
+Finally, expose the host and make it available to Weave Net by setting its DNS record:
```

Comment:
we don't "expose by setting a DNS record", we really "expose and set a DNS record"...